whois-nrtm4/src/main/java/net/ripe/db/nrtm4/dao/NrtmVersionInfoRepository.java

```diff
@@ -32,9 +31,11 @@ public class NrtmVersionInfoRepository {
             NrtmDocumentType.valueOf(rs.getString(5)),
             rs.getInt(6)
         );
+    private final String versionColumns = "v.id, src.name, v.version, v.session_id, v.type, v.last_serial_id ";
```

Comment:
Do you need to declare a field for these? It's only used twice and makes the statement less readable.